odoo/addons/base/controllers/avatar.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from hashlib import sha1
+import base64
+
+from odoo.http import Controller, request, route, Response, local_redirect
+from odoo.modules.module import get_module_resource
+
+class Avatar(Controller):
+    @route(['/web/avatar/<int:user_id>', '/web/avatar/<int:user_id>/<string:image_field>'], auth='public', cors="*", methods=['GET'])
```

Comment:
warning, you sesactivated the authentication to your endpoint and desactivated cors...
Don't forget to put back all the security mechanisms...